webxdc.d.ts

```diff
@@ -58,6 +71,23 @@ interface Webxdc<T> {
    * @param description short, human-readable description what this update is about. this is shown eg. as a fallback text in an email program.
    */
   sendUpdate(update: SendingStatusUpdate<T>, description: string): void;
+  /**
+   * Send a message with file, text or both to a chat.
+   * Asks user to what Chat to send the message to.
+   * Always exits the xdc, please save your app state before calling this function
+   * @param content
+   * @returns returns a promise that never resolves (because the xdc closes), but is rejected on error.
+   */
+  sendToChat(content: sendOptions): Promise<void>;
```

Comment:
shouldn't be `Promise<boolean>` ? so false is returned if there was an error?

Reply:
If there was an error, then the promise is rejected instead of resolved, so there is no need for a boolean. this is not rust were a future returns a result, in js all futures have a result type built in basically.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise